src/Microsoft.Framework.PackageManager/project.json

```diff
@@ -43,9 +43,9 @@
 
     "scripts": {
         "postbuild": [
-            "%project:Directory%/../../build/batchcopy %project:BuildOutputDir%/Debug/aspnet50/*.* %project:Directory%/../../artifacts/build/kre-clr-win-x86/bin/lib/Microsoft.Framework.PackageManager",
-            "%project:Directory%/../../build/batchcopy %project:BuildOutputDir%/Debug/aspnet50/*.* %project:Directory%/../../artifacts/build/kre-mono/bin/lib/Microsoft.Framework.PackageManager",
-            "%project:Directory%/../../build/batchcopy %project:BuildOutputDir%/Debug/aspnetcore50/*.* %project:Directory%/../../artifacts/build/kre-coreclr-win-x86/bin/lib/Microsoft.Framework.PackageManager"
+            "%project:Directory%/../../build/batchcopy %project:BuildOutputDir%/Debug/aspnet50/*.* %project:Directory%/../../artifacts/build/dnx-clr-win-x86/bin/lib/Microsoft.Framework.PackageManager",
+            "%project:Directory%/../../build/batchcopy %project:BuildOutputDir%/Debug/aspnet50/*.* %project:Directory%/../../artifacts/build/dnx-mono/bin/lib/Microsoft.Framework.PackageManager",
+            "%project:Directory%/../../build/batchcopy %project:BuildOutputDir%/Debug/aspnetcore50/*.* %project:Directory%/../../artifacts/build/dnx-coreclr-win-x86/bin/lib/Microsoft.Framework.PackageManager"
         ]
```

Comment:
Not yet, the runtime doesn't understand them or build them. That will happen in an upcoming PR, let's get this one in first